botfront/imports/api/nlu_model/nlu_model.methods.js

```diff
@@ -26,6 +29,34 @@ const gazetteDefaults = {
 };
 
 Meteor.methods({
+    async 'nlu.canonicalizeExamples'(items, modelId) {
```

Comment:
Since `nlu.canoicalizeExamples` is only called on the server, does it need to be a Meteor method? Can't it just be a regular async function that runs in the server?

Reply:
Changed to be a utils function, thanks for the review!